src/test/java/utils/PropertiesReader.java

```diff
@@ -0,0 +1,54 @@
+package utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertiesReader {
+    static private PropertiesReader instance;
+    //Variables for the properties
+    private String urlEndpoint = null;
+    private String authorizationString = null;
```

Comment:
Please remove these attributes and use directly the methods that you already created